app/src/lib/api.ts

```diff
@@ -554,7 +553,7 @@ export async function fetchUser(endpoint: string, token: string): Promise<Accoun
     emails = [ ]
   }
 
-  return new Account(user.login, endpoint, token, emails, user.avatarUrl, user.id, user.name)
+  return new Account(user.login, endpoint, token, emails, user.avatar_url, user.id, user.name)
```

Comment:
I am not sure why this `camelCase` field still worked even with the camel-case plugin explicitly disabled. 

But, in order to provide a TypeScript definition file I opted to define the Responses in the same format as the GitHub API.